Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -309,9 +315,13 @@ try {
                     $srcContent = Get-ContentLF -Path $srcFile
                 }
 
-
                 $dstFile = Join-Path $dstFolder $fileName
-                if (Test-Path -Path $dstFile -PathType Leaf) {
+                if ($unusedALGoSystemFiles -contains $fileName) {
+                    if (Test-Path -Path $dstFile -PathType Leaf) {
```

Comment:
This check is duplicated below.

Consider moving `$unusedALGoSystemFiles -contains $fileName` as a subcheck instead.

Reply:
Moving that to a subcheck - then I would have to check for the unusedALGoSystemFiles before adding the new file to updateFiles. Will make a small update and add a comment